package.json

```diff
@@ -80,32 +80,32 @@
     "inputmask": "5.0.7",
     "ng2-charts": "3.0.11",
     "ngx-mat-select-search": "4.2.0",
-    "rxjs": "7.5.5",
+    "rxjs": "~7.5.0",
     "tslib": "2.4.0",
     "zone.js": "0.11.4"
```

Comment:
suggestion: remember to update the `NOTICE.md` file with the updated version of production dependencies